packages/kokkos/core/src/impl/Kokkos_Atomic_Compare_Exchange_Strong.hpp

```diff
@@ -328,11 +328,13 @@ atomic_compare_exchange(volatile T* const, const Kokkos::Impl::identity_t<T>,
 }
 #endif
 
+#ifndef _WIN32
```

Comment:
What is the config here and why do we need this change? We are currently testing Kokkos with Visual Studio during PR testing and it passes. I'd prefer if we could fix this in Kokkos first and make sure we support the config.